pkg/publicapi/endpoint/agent/endpoint.go

```diff
@@ -53,6 +53,7 @@ func NewEndpoint(params EndpointParams) (*Endpoint, error) {
 	g.GET("/debug", e.debug)
 	g.GET("/config", e.config)
 	g.GET("/license", e.license)
+	g.GET("/authconfig", e.nodeAuthConfig)
```

Comment:
you might want to verify how some of the commands will work against a compute node. Meaning calling `bacalhau agent version` for example against a compute only node

Reply:
Integration Test Added